release/src/main/groovy/MoblieGamingJavaUtils.groovy

```diff
@@ -0,0 +1,176 @@
+import java.text.SimpleDateFormat
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+class MobileGamingJavaUtils {
+
+  public static final RUNNERS = [DirectRunner: "direct-runner",
+    DataflowRunner: "dataflow-runner",
+    SparkRunner: "spark-runner",
+    ApexRunner: "apex-runner",
+    FlinkRunner: "flink-runner"]
+
+  public static final EXECUTION_TIMEOUT = 20
+
+  // Lists used to verify team names generated in the LeaderBoard example
+  public static final COLORS = new ArrayList<>(Arrays.asList(
+    "Magenta",
+    "AliceBlue",
+    "Almond",
+    "Amaranth",
+    "Amber",
+    "Amethyst",
+    "AndroidGreen",
+    "AntiqueBrass",
+    "Fuchsia",
+    "Ruby",
+    "AppleGreen",
+    "Apricot",
+    "Aqua",
+    "ArmyGreen",
+    "Asparagus",
+    "Auburn",
+    "Azure",
+    "Banana",
+    "Beige",
+    "Bisque",
+    "BarnRed",
+    "BattleshipGrey"))
+
+  private static final USERSCORE_OUTPUT_PREFIX = "java-userscore-result-"
+
+  private static final HOURLYTEAMSCORE_OUTPUT_PREFIX = "java-hourlyteamscore-result-"
+
+  public static String generateCommand(String exampleName, String runner, TestScripts t){
+    String commonArgs = "--tempLocation=gs://${t.gcsBucket()}/tmp --runner=${runner} "
+
+    if(exampleName.equals("UserScore")){
+      return generateUserScoreCommand(runner, t, commonArgs)
+    } else if(exampleName.equals("HourlyTeamScore")){
+      return generateHourlyTeamScoreCommand(runner, t, commonArgs)
+    } else if(exampleName.equals("LeaderBoard")){
+      return generateLeaderBoardCommand(runner, t, commonArgs)
+    } else if(exampleName.equals("GameStats")){
+      return generateGameStatsCommand(runner, t, commonArgs)
+    } else if(exampleName.equals("Injector")){
+      return generateInjectorCommand(t)
+    }
+    return "ERROR: Not found the example ${exampleName}."
+}
+
+  public static String getUserScoreOutputName(String runner){
+    return "${USERSCORE_OUTPUT_PREFIX}${RUNNERS[runner]}.txt"
+  }
+
+  public static String getHourlyTeamScoreOutputName(String runner){
+    return "${HOURLYTEAMSCORE_OUTPUT_PREFIX}${RUNNERS[runner]}.txt"
+  }
+
+
+  private static String generateUserScoreCommand(String runner, TestScripts t, String commonArgs){
```

Comment:
I find the use of `commonArgs` as a parameter a bit skeevy - perhaps a method to generate them instead?

It's especially true as the `cmd` is built basically the same way in four places - add a similar method for configuring that command?